src/layouts/components/ContactUs/ContactCard.tsx

```diff
@@ -232,7 +232,6 @@ export const ContactCard = ({
 
         <Button
           variant="clear"
-          w="100%"
```

Comment:
don't we need this for full-width button?

or is it that the button is *not* supposed to be full-width?

Reply:
the button is **not** supposed to be full width, clarified with designer